Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM ubuntu:14.04
 MAINTAINER PhenoMeNal-H2020 Project <phenomenal-h2020-users@googlegroups.com>
-LABEL Description="Galaxy test for running inside Kubernetes."
+
+LABEL Description="Galaxy 16.07-phenomenal for running inside Kubernetes."
+LABEL software="Galaxy"
+LABEL software.version="16.07-pheno"
+LABEL version="0.1"
```

Comment:
These LABEL statements could be compressed into a single one, reducing the number of layers in the container.